client/src/components/MoreInfo.js

```diff
@@ -1,7 +1,95 @@
 import React from 'react'
+import {Segment} from "semantic-ui-react";
 
-class MoreInfo extends React.Component {
-    render() {
-        return <h1>MoreInfo</h1>
-    }
-} export default MoreInfo
\ No newline at end of file
+const MoreInfo = () => (
+    <div>
+        <Segment>
+            <h1>
+                Information About Food Deserts:
+            </h1></Segment>
+
+        <Segment color='green'>
+            <h3> What Is The Official Definition Of A Food Desert? </h3>
```

Comment:
I think sentence case ("What is the official definition of a food desert?") may look better for the headers on this page